extras/ams_wrapper/README.md

```diff
@@ -0,0 +1,386 @@
+attributes and labels. The config can also specify input data – like resolution or pre-processing parameters. 
+
+
+### Object detection
+
+In this category, models should return response in the shape `[1, 1, N, 7]` where N is the number of detected bounding boxes.
+For each detection, the description has the 
+format: [image_id, label, conf, x_min, y_min, x_max, y_max], where:
+- image_id - ID of the image in the batch
+- label - predicted class ID
+- conf - confidence for the predicted class
+- (x_min, y_min) - coordinates of the top left bounding box corner
+- (x_max, y_max) - coordinates of the bottom right bounding box corner.
+
+There are several Object Detection models available in the [OpenVINO Model Zoo](https://docs.openvinotoolkit.org/2020.2/_models_intel_index.html). 
+The Inference Server Docker image comes pre-built with the following Object Detection models: 
+
+* vehicle-detection - [vehicle-detection-adas-binary-0001](https://github.com/opencv/open_model_zoo/tree/master/models/intel/vehicle-detection-adas-binary-0001)
+* face-detection - [face-detection-adas-0001](https://github.com/opencv/open_model_zoo/tree/master/models/intel/face-detection-adas-binary-0001)
+* person-vehicle-bike-detection-crossroad - [person-vehicle-bike-detection-crossroad-0078](http://•%09https:/github.com/opencv/open_model_zoo/tree/master/models/intel/person-vehicle-bike-detection-crossroad-0078)
+
+Each model should include also a configuration file in json format. Example of such
+configuration file is [here](ams_models/vehicle_detection_adas_model.json)
+
+Below is a sample of the model output:
+```json
+{
+   "subtype" : "vehicleDetection",
+   "entities" : [
+      {
+         "box" : {
+            "h" : 0.10686594247818,
+            "t" : 0.336242735385895,
+            "w" : 0.130945563316345,
+            "l" : 0.789495408535004
+         },
+         "tag" : {
+            "confidence" : 0.999798119068146,
+            "value" : "vehicle"
+         }
+      },
+      {
+         "box" : {
+            "h" : 0.143958985805511,
+            "l" : 0.640124976634979,
+            "t" : 0.308264225721359,
+            "w" : 0.159151077270508
+         },
+         "tag" : {
+            "value" : "vehicle",
+            "confidence" : 0.572816431522369
+         }
+      },
+    ],
+   "type" : "entity"
+}
+```
+
+### Classification models
+
+Classification models, in this category, give the results in softmax layer. This include a set
+of probabilities in classes defined for the model.
+Each output of the model should have the shape `[1, C , ...]`. First dimension represent the batch size,
+which should be set to 1. `C` represent all classes defined in the model. Remaining dimensions 
+are ignored (if present, first index is used).
+
+Examples of such models are available in the [OpenVINO Model Zoo](https://docs.openvinotoolkit.org/2020.2/_models_intel_index.html).
+In OVMS docker image for LVA is included the following classification model: 
```

Comment:
The following classification model is included in the OVMS docker image for LVA: